bindings/golang/zenroom/zenroom.go

```diff
@@ -3,86 +3,29 @@
+//     return exec(script, conf, data, keys, stdout, stdout_size, stderr, stderr_size);
+// }
+import "C"
 
 import (
-	"fmt"
 	"unsafe"
-
-	// dummy import to include binary in dependencies
-	_ "github.com/dyne/Zenroom/bindings/golang/zenroom/lib"
 )
 
 // maxString is zenroom defined buffer MAX_STRING size
-const maxString = 1048576
-
-// config is an unexported config struct used to handle the functional variadic
-// configuration options defined below.
-type config struct {
-	Keys      []byte
-	Data      []byte
-	Conf      string
-	Verbosity int
-}
+const BUFSIZE = 2 * 1024 * 1024
 
-// Option is a type alias we use to supply optional configuration to the primary
-// Exec method this library exposes. Option is a functional configuration type
-// alias for a function that takes as a parameter an unexported configuration
-// object. Callers can supply keys, data, conf and verbosity via an Option using
-// one of the WithXXX helper functions.
-type Option func(*config)
-
-// WithKeys is a configuration helper that allows the caller to pass in a value
-// for the KEYS parameter that Zenroom supports. The value of KEYS is typically
-// a string, often JSON formatted, that contains one or more keys that the
-// primary script requires in order to operate. These are passed separately from
-// the main script as they will typically have different security requirements
-// than the main script contents. Keys must be passed in to the helper in a byte
-// slice.
-func WithKeys(keys []byte) Option {
-	return func(c *config) {
-		c.Keys = keys
-	}
-}
-
-// WithData is a configuration helper that allows the caller to pass in a value
-// for the DATA parameter that Zenroom supports. The value of KEYS is a string,
-// often but not required to be JSON formatted, containing data over which the
-// script should operate. As with the KEYS property, DATA is passed separately
-// from either the SCRIPT or the KEYS as these different values will often have
-// different security requirements or may come from different sources. DATA must
-// be passed as a byte slice.
-func WithData(data []byte) Option {
-	return func(c *config) {
-		c.Data = data
-	}
-}
-
-// WithConf is a configuration helper that allows the caller to pass in a value
-// for the CONF parameter that Zenroom supports. The default for this value if
-// not supplied is an empty string.
-func WithConf(conf string) Option {
-	return func(c *config) {
-		c.Conf = conf
-	}
-}
-
-// WithVerbosity is a configuration helper that allows the caller to specify how
-// verbose Zenroom should be with its output. The value of this configuration
-// must be an integer from 1 to 3, where 1 is the least verbose, and 3 is the
-// most. The default if this value is not supplied is 1.
-func WithVerbosity(verbosity int) Option {
-	return func(c *config) {
-		c.Verbosity = verbosity
-	}
+type ZenResult struct {
+    output string;
+    err string;
```

Comment:
Would prefer to rename it in logs to keep it consistent with js. Also actually are the output of the execution, not just errors... what you think?

Reply:
I agree with you about using `logs` over `err`. I have used the same naming as python as I didn't know that js used `logs`.